app/components/plans/treatmentRows.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
Is this an old file? Looks like it's something temporary that needs to be deleted as part of a clean-up commit.